packages/frontend/src/components/staking/components/Validators.js

```diff
@@ -38,12 +42,16 @@ export default function Validators({ validators, stakeFromAccount }) {
                 </div>
             )}
             <ListWrapper>
-                {validators.filter(v => v.accountId.includes(validator)).map((validator, i) => 
-                    <ValidatorBox
-                        key={i}
-                        validator={validator}
-                    />
-                )}
+                {validators.filter(v => v.accountId.includes(validator))
+                    .sort((first, second) =>
```

Comment:
Ideally we would abstract this into a selector so that it's memoized and does not resort on re-render

Reply:
To do so, we should move the validator and setValidator to store.
Otherwise, create a callback with params and use it in the component so it shouldn't work as planned
Could we process with useMemo in this case?

Reply:
Agreed, I think `useMemo` works well here since we use references to local state 👍

Reply:
Agreed, I think useMemo works well here since we use references to local state 👍